src/logic/small.lean

```diff
@@ -63,11 +63,10 @@ open_locale classical
 
 theorem small_congr {α : Type*} {β : Type*} (e : α ≃ β) : small.{w} α ↔ small.{w} β :=
```

Comment:
I think a better way to golf this would be to add
```lean
theorem small_map {α : Type*} {β : Type*} (e : α ≃ β) [small.{w} α] : small.{w} β :=
```
And then use that lemma as
```suggestion
theorem small_congr {α : Type*} {β : Type*} (e : α ≃ β) : small.{w} α ↔ small.{w} β :=
⟨small_map e, small_map e.symm⟩
```

Reply:
I bet most of the uses of `small_congr` below can them be replaced with `small_map` to save a line

Reply:
I've pushed that change. Seems to have worked.